x-pack/test/reporting_api_integration/reporting_and_security/generate_csv_discover.ts

```diff
@@ -6,73 +6,72 @@
  */
 
 import expect from '@kbn/expect';
-import supertest from 'supertest';
-import { JOB_PARAMS_RISON_CSV_DEPRECATED } from '../services/fixtures';
+import { SearchSourceFields } from 'src/plugins/data/common';
+import { ReportApiJSON } from '../../../plugins/reporting/common/types';
 import { FtrProviderContext } from '../ftr_provider_context';
 
 // eslint-disable-next-line import/no-default-export
 export default function ({ getService }: FtrProviderContext) {
-  const esArchiver = getService('esArchiver');
-  const supertestSvc = getService('supertest');
   const reportingAPI = getService('reportingAPI');
 
-  const generateAPI = {
-    getCsvFromParamsInPayload: async (jobParams: object = {}) => {
-      return await supertestSvc
-        .post(`/api/reporting/generate/csv`)
-        .set('kbn-xsrf', 'xxx')
-        .send(jobParams);
-    },
-    getCsvFromParamsInQueryString: async (jobParams: string = '') => {
-      return await supertestSvc
-        .post(`/api/reporting/generate/csv?jobParams=${encodeURIComponent(jobParams)}`)
-        .set('kbn-xsrf', 'xxx');
-    },
-  };
+  describe('Generate CSV from SearchSource', () => {
+    // skipping this to avoid shard failures
+    it.skip(`Searches large amount of data`, async () => {
```

Comment:
I created this test as a way to reproduce the shard failures and see the warnings in the UI.

It can be unskipped if ES improves the shard failures in the `_scroll` API or if CSV export switches to PIT for this kind of search.